spec/services/idv/phone_step_spec.rb

```diff
@@ -128,6 +133,18 @@
       expect { subject.submit(phone: fail_phone) }.to_not change { throttle.attempts }
     end
 
+    it 'logs a throttled attempts_tracker event' do
+      while throttle.attempts <= Throttle.max_attempts(:proof_address) - 1
+        throttle.increment!
+      end
+
+      subject.submit(phone: bad_phone)
+      expect(@irs_attempts_api_tracker).to receive(:idv_phone_otp_sent_rate_limited)
+      _result = subject.async_state_done(subject.async_state)
```

Comment:
if we're going to ignore the result, maybe don't even assign it?
```suggestion
      subject.async_state_done(subject.async_state)
```